assets/src/block-validation/error.js

```diff
@@ -0,0 +1,289 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useDispatch, useSelect } from '@wordpress/data';
+import { Button, PanelBody } from '@wordpress/components';
+import { sprintf, __ } from '@wordpress/i18n';
+import { useMemo } from '@wordpress/element';
+import { addQueryArgs } from '@wordpress/url';
+import { BlockIcon } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import { useInlineData } from './use-inline-data';
+import {
+	AMP_VALIDITY_REST_FIELD_NAME,
+	VALIDATION_ERROR_ACK_ACCEPTED_STATUS,
+	VALIDATION_ERROR_ACK_REJECTED_STATUS,
+	VALIDATION_ERROR_NEW_ACCEPTED_STATUS,
+	VALIDATION_ERROR_NEW_REJECTED_STATUS,
+} from './constants';
+import { NewTabIcon } from './icon';
+
+/**
+ * Component rendering an icon representing JS, CSS, or HTML.
+ *
+ * @param {Object} props
+ * @param {string} props.type The error type.
+ */
+function ErrorTypeIcon( { type } ) {
```

Comment:
I'd suggest moving all SVG icons here into actual separate `.svg` files

1. Improves readability
1. Allows webpack svg loader to optimize SVGs upon bundling

Reply:
Done in cfe19d7fd